lib/lita/handlers/google_images.rb

```diff
@@ -21,11 +21,18 @@ class GoogleImages < Handler
         "image QUERY" => "Displays a random image from Google Images matching the query."
       })
 
-      def fetch(response)
+      route(/(?:animate|gif)(?:\s+me)? (.+)/i, :fetch_anim, command: true, help: {
+        "animate QUERY" => "Displays a random animation from Google Images matching the query."
+      })
+
+      def fetch_anim(response)
+        fetch(response, animated = true)
```

Comment:
I think this should just be `fetch(response, true)`. (Same for the call in `fetch` below.)